### PR TITLE
msx1_cass.xml: Added 70 items, 68 working. Replaced 2 items. Removed 2 items. Renamed 1 item.

### PR DESCRIPTION
**Pull request by wilbertpol**
Replaced Booty (Europe) with a better dump. [file-hunter] 
Replaced North Sea Helicopter with a better dump. [file-hunter]

Removed Silicon Dreams (Europe, alt 2) and Silicon Dreams (Europe, alt 3).

Renamed Soviet (Europe?) to Soviet (Spain).

New working software list items
-------------------------------
Inleiding tot de SpectraVideo SV 728 (Netherlands) [file-hunter] 
TEACH YOURSELF BASIC (United Kingdom) [file-hunter] 
The Archers (Europe) [file-hunter]
Bang! Bang! (Japan) [file-hunter]
The Growing Pains of Adrian Mole (United Kingdom) [file-hunter] 
The Munsters (Europe) [file-hunter]
River Chase (Japan) [file-hunter]
Scapeghost (Europe) [file-hunter]
Sea Hunter (Europe) [file-hunter]
The Secret Diary of Adrian Mole Aged 13¾ (Europe) [file-hunter] 
Silicon Dreams I - Snowball (United Kingdom) [file-hunter] 
Silicon Dreams II - Return to Eden (United Kingdom) [file-hunter] 
Silicon Dreams III - The Worm in Paradise (United Kingdom) [file-hunter] 
Sky Diver (Japan, alt) [file-hunter]
Soviet (Spain, alt) [file-hunter]
Space Harrier II (Europe) [file-hunter]
Starbuster (Europe) [file-hunter]
Supermaze (United Kingdom) [file-hunter]
Tank (Finland) [file-hunter]
Tape Login MSX Game Book (Japan) [file-hunter]
The Temptation of the Apartment Wife (Japan) [file-hunter] 
Time and Magik III - The Price of Magik (Europe) [file-hunter] 
Time and Magik (United Kingdom) [file-hunter]
Time and Magik (United Kingdom, cracked) [file-hunter] 
Time and Magik (United Kingdom, alt) [file-hunter] 
The Tomato Princess from Salad Land (Japan) [file-hunter] 
Vampire (Europe) [file-hunter]
Vampire (Europe, alt) [file-hunter]
Video Poker (Brazil, Disprosoft) [file-hunter]
W Series 1 - Biotech / Killer Station (Japan) [file-hunter] 
W Series 3 - Akarui Nouen / Fire Ball (Japan) [file-hunter] 
W Series 4 - Super Doors / Busy Rainy Day (Japan) [file-hunter] 
Wilco (Spain) [file-hunter]
Wild Cat (Japan) [file-hunter]
Color Plotter/Printer-you yō Sekai Chizu Soft (Japan) [file-hunter] 
The Wreck (United Kingdom) [file-hunter]
Xenon (United Kingdom) [file-hunter]
Zero Fighter (Japan) [file-hunter]
The Queen's Footsteps (Italian) [Davide Bucci]
Silvestro (Italy) [file-hunter]
Torreoscura (Spanish) [file-hunter]
Super MSX N.1 (1987-05) (Italy) [file-hunter]
Super MSX N.2 (1987-06) (Italy) [file-hunter]
Super MSX N.3 (1987-07/08) (Italy) [file-hunter]
Super MSX N.4 (1987-09) (Italy) [file-hunter]
Super MSX N.5 (1987-10) (Italy) [file-hunter]
Super MSX N.6 (1987-11) (Italy) [file-hunter]
Super MSX N.7 (1987-12) (Italy) [file-hunter]
Super MSX N.8 (1988-01) (Italy) [file-hunter]
Super MSX N.9 (1988-02) (Italy) [file-hunter]
Super MSX N.10 (1988-03) (Italy) [file-hunter]
Super MSX N.11 (1988-04) (Italy) [file-hunter]
Super MSX N.12 (1988-05) (Italy) [file-hunter]
Super MSX N.13 (1988-06) (Italy) [file-hunter]
Super MSX N.14 (1988-09) (Italy) [file-hunter]
Super MSX N.15 (1988-10) (Italy) [file-hunter]
Super MSX N.16 (1988-11) (Italy) [file-hunter]
Super MSX N.17 (1988-12) (Italy) [file-hunter]
Super MSX N.18 (1989-01) (Italy) [file-hunter]
Super MSX N.19 (1989-02) (Italy) [file-hunter]
Super MSX N.20 (1989-04) (Italy) [file-hunter]
Super MSX N.21 (1989-05) (Italy) [file-hunter]
Super MSX N.22 (1989-06) (Italy) [file-hunter]
Super MSX N.23 (1989-09) (Italy) [file-hunter]
Super MSX N.24 (1989-10) (Italy) [file-hunter]
Super MSX N.25 (1989-12) (Italy) [file-hunter]
Super MSX N.26 (1990-01) (Italy) [file-hunter]
Super MSX N.27 (1990-02) (Italy) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
Light Pen Graphic v1.0 (Japan) [file-hunter]
The Queen's Footsteps (English) [Davide Bucci]

### CODE REVIEW

File: hash/msx1_cass.xml [hunk 1]
```diff
@@ -143,6 +143,56 @@ license:CC0-1.0
...
+		</part>
+		<part name="cass1b" interface="msx_cass">
+			<dataarea name="cass" size="16800">
+				<rom name="light pen graphic v1.0 - sanyo (side b).cas" size="16800" crc="f8aad1f1" sha1="45a254be32167d2722a50355b673ea39cde74892"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="teachysb">
+		<description>Teach Yourself Basic (United Kingdom)</description>
```
Comment (cuavas) [inline hash/msx1_cass.xml:179]: “BASIC” is an acronym – it should be in uppercase.

the title of the software was all caps on the manual: https://www.computinghistory.org.uk/det/12628/Teach-Yourself-BASIC/

File: hash/msx1_cass.xml [hunk 2]
```diff
@@ -12989,6 +13126,28 @@ license:CC0-1.0
...
+		<part name="cass2" interface="msx_cass">
+			<dataarea name="cass" size="60407">
+				<rom name="scapeghost - level 9 computing (part 2).cas" size="60407" crc="24375acf" sha1="90d08b32b264bd6fcf81794d973d4dbfe0a8f2c5"/>
+			</dataarea>
+		</part>
+		<part name="cass3" interface="msx_cass">
+			<dataarea name="cass" size="60407">
+				<rom name="scapeghost - level 9 computing (part 3).cas" size="60407" crc="f6fdb73d" sha1="58d8c07b8f90c7456ea52610f801e5228830b049"/>
+			</dataarea>
+		</part>
```
Comment (cuavas) [inline hash/msx1_cass.xml:13148]: Do we have labels for these cassettes that can go in the `part_id`?

File: hash/msx1_cass.xml [hunk 3]
```diff
@@ -13524,6 +13712,18 @@ license:CC0-1.0
...
+		<description>Sky Diver (Japan, alt)</description>
+		<year>1984</year>
+		<publisher>SoftBank</publisher>
+		<info name="usage" value="Load with BLOAD&quot;CAS:&quot;,R."/>
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="11732">
+				<rom name="sky diver - hudson soft (alt).cas" size="11732" crc="30522dd2" sha1="96873d866b7989d76964db1aaedeec13a6c9568b"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx1_cass.xml:13725]: Is this actually from SoftBank?  The title screen says Hudson Soft.  I believe at the time SoftBank had exclusive distribution rights to Hudson’s MSX software.

File: hash/msx1_cass.xml [hunk 4]
```diff
@@ -16286,6 +16764,67 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="wseries1">
+		<description>W Series 1 - Biotech / Killer Station (Japan)</description>
+		<year>1983</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="alt_title" value="バイオテック／キラーステーション"/>
```
Comment (cuavas) [inline hash/msx1_cass.xml:16771]: Japanese language does not use slashes like this.  Also, the case insert does say “Wシリーズ1”: https://images.generation-msx.nl/cover/07778259.jpg

File: hash/msx1_cass.xml [hunk 5]
```diff
@@ -16286,6 +16764,67 @@ license:CC0-1.0
...
+				<rom name="w series 1 - hudson soft (side b).cas" size="17088" crc="3cb0c6db" sha1="614b87a4a2abf8a452d7747e876bbaa23708ed1a"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="wseries3">
+		<description>W Series 3 - Akarui Nouen / Fire Ball (Japan)</description>
+		<year>1983</year>
+		<publisher>Hudson Soft</publisher>
+		<info name="alt_title" value="明るい農園／ファイアーボール"/>
```
Comment (cuavas) [inline hash/msx1_cass.xml:16792]: I haven’t checked these, but I assume they’re similar to W Series 1.

File: hash/msx1_cass.xml [hunk 6]
```diff
@@ -16786,7 +17351,53 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="wldmscpp">
+		<description>World Map Software for Color Plotter Printer (Japan)</description>
+		<year>1984</year>
+		<publisher>National</publisher>
+		<info name="alt_title" value="カラープロッタプリンタ用世界地図ソフト"/>
```
Comment (cuavas) [inline hash/msx1_cass.xml:17358]: Please don’t translate titles for descriptions – just use “Color Plotter/Printer-you yō Sekai Chizu Soft”.

File: hash/msx1_cass.xml [hunk 7]
```diff
@@ -17026,6 +17651,19 @@ license:CC0-1.0
 
 <!-- Doujin -->
 
+	<software name="chiteidb">
+		<description>Chitei Daibouken (Japan)</description>
+		<year>1987</year>
+		<publisher>&lt;doujin&gt;</publisher>
+		<info name="developer" value="Yasunori Takahara"/>
```
Comment (cuavas) [inline hash/msx1_cass.xml:17658]: The Japanese title is “地底大冒険”.

File: hash/msx1_cass.xml (hunk 2 above)
Comment (wilbertpol) [reply hash/msx1_cass.xml:13148]: I have not found pictures of any cassettes for the MSX version so I did not set the part_ids.

File: hash/msx1_cass.xml (hunk 3 above)
Comment (wilbertpol) [reply hash/msx1_cass.xml:13725]: It was indeed from Hudson Soft

File: hash/msx1_cass.xml [hunk 8]
```diff
@@ -16786,10 +16786,10 @@ Side B
 	</software>
 
 	<software name="wseries3">
-		<description>W Series 3 - Akarui Nouen / Fire Ball (Japan)</description>
+		<description>W Series 3 - Fire Ball / Akarui Nouen (Japan)</description>
 		<year>1983</year>
 		<publisher>Hudson Soft</publisher>
-		<info name="alt_title" value="Wシリーズ3"/>
+		<info name="alt_title" value="Wシリーズ3　ファイアボール・明るい農園"/>
```
Comment (cuavas) [inline hash/msx1_cass.xml:16792]: Covers exist listing Fire Ball first or listing Akarui Nouen first.  I don’t know which are earlier.

**Comment by cuavas** [comment]
I merged this, but the two items you listed as not working (Light Pen Graphic v1.0 (Japan) and The Queen's Footsteps (English)) are both marked as partially supported rather than unsupported.  Can you please confirm whether this is correct?

**Comment by wilbertpol** [comment]
Yes, for both of those only a part of the program(s) work. For the Queen's Footsteps only the first part of the software can be loaded, not the rest. For the Light Pen Graphic only some demo programs work the most important program for drawing with a light pen is not working.

Should partially supported be listed in a separate section then?

**Comment by cuavas** [comment]
> Yes, for both of those only a part of the program(s) work. For the Queen's Footsteps only the first part of the software can be loaded, not the rest. For the Light Pen Graphic only some demo programs work the most important program for drawing with a light pen is not working.

If key features of the software do not work **the software is to be marked unsupported**.  I *know* I’ve mentioned this before.

Marking software as partially supported is for cases where it would have “yellow” warnings for a system (e.g. a game that can be played to completion but has graphical and/or sound glitches that don’t break gameplay), or software with multiple distinct pieces of functionality where the majority of the features function correctly (e.g. a productivity suite where full-screen presentation mode doesn’t work but the rest of the functionality is usable).

If only demos are working for the lightpen software, that’s not supported by any meaningful definition.  The primary functionality of the software is not working.  It is not supported.

**Comment by wilbertpol** [comment]
Thanks, I'll update the supported flags for those entries on my next PR then.

**Comment by imulilla** [comment]
This version of "Queen's Footsteps" includes only a loader and the 3 parts blocks, the cassette does not stop when the game finishes loading and I have not been able to load the others, the author comments that it has a password system but I can't find a way to enter them. I think it is better to add the version that includes 3 CAS files

**Pull request merged**